pkg/platform/model/buildplanner.go

```diff
@@ -305,20 +349,22 @@ func (bp *BuildPlanner) StageCommit(params StageCommitParams) (strfmt.UUID, erro
 }
 
 func (bp *BuildPlanner) GetBuildExpression(owner, project, commitID string) (*buildexpression.BuildExpression, error) {
-	resp := &bpModel.BuildPlan{}
-	err := bp.client.Run(request.BuildExpression(owner, project, commitID), resp)
+	logging.Debug("GetBuildExpression")
```

Comment:
Drop

Reply:
This and other logging calls with function names could still be useful when tracing through logs, no?

Reply:
Ok yeah, if you think it's useful. Possibly add the input to the debug call? I assumed this was a local debugging statement, hence why I flagged it.